### PR TITLE
Revise media handling

### PR DESCRIPTION
**Pull request by thetkpark**
- Use ULID for owner image naming in GCS bucket
- Publish message to PubSub for guest media
- Create Google Drive folder and shared link for graduate on onboarding

Screenshot 2566-10-28 at 13 35 05
Screenshot 2566-10-28 at 13 36 00
Screenshot 2566-10-28 at 13 36 13

### CODE REVIEW

**Comment by GGolfz** [comment]
So, we need to have subscriber server in order to process image upload also?

**Comment by GGolfz** [comment]
How do we gonna handle the google drive link for graduate? New command in line text message?

**Comment by thetkpark** [comment]
> How do we gonna handle the google drive link for graduate? New command in line text message?

yesๆ

File: backend/src/webhook/eventHandler/MessageHandlerFactory.ts [hunk 1]
```diff
@@ -51,7 +51,15 @@ export class MessageHandlerFactory {
 			],
 			[
 				this.getKey(false, MessageType.Image),
-				new GuestImageMessageHandler(
+				new GuestContentMessageHandler(
```
Comment (GGolfz) [inline backend/src/webhook/eventHandler/MessageHandlerFactory.ts]: Better to initialize only once?

**Review by GGolfz — APPROVED** [review]
LGTM

**Pull request merged**